docs/autostart_systemd.rst

```diff
@@ -24,7 +24,7 @@ Then create the new service file:
 
 :code:`sudo -e /etc/systemd/system/red@.service`
 
-Paste the following and replace all instances of :code:`username` with the username, and :code:`path` with the location you obtained above:
+Paste the following and replace all instances of :code:`username` with your Linux username (you can use the :code:`whoami` command if you're unsure), and :code:`path` with the location you obtained above:
```

Comment:
This is imo a little unclear on *why* would user want to use this command, I think adding something like "if you're unsure what your username is" would be enough here, but anything that makes it more clear when you should use the command and/or what is the reader "unsure" of is gonna work here so if you think something would fit better here, you can of course put that instead ;)
```suggestion
Paste the following and replace all instances of :code:`username` with your Linux username (you can use the :code:`whoami` command if you're unsure what your username is), and :code:`path` with the location you obtained above:
```

Reply:
I decided to rewrite that section entirely since the whole thing was a little unclear at a couple points.